### PR TITLE
Refactor select query load test directory name and add load test trigger workflow

### PR DESCRIPTION
**Pull request by kaneeldias**
## Purpose
$subject 

Current load test failed due to the directory name (test name) being different from the ballerina project name in Ballerina.toml

Related issue: https://github.com/ballerina-platform/ballerina-standard-library/issues/2270

### CODE REVIEW

File: .github/workflows/invoke-load-tests.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name: Invoke Load Tests
```
Comment (shafreenAnfar) [inline .github/workflows/invoke-load-tests.yml:1]: We just decided to change the name as `Trigger Load Tests`

File: .github/workflows/invoke-load-tests.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+on:
+  workflow_dispatch:
+    inputs:
+      tests:
+        description: >
+          Array of test names
+          Example-["mysql_select", "mysql_insert"]
```
Comment (shafreenAnfar) [inline .github/workflows/invoke-load-tests.yml:9]: We just decided to change the description to,

```
List of test names

This needs to be filled only if you want to run a specific set of tests. Example: [“foo”, “bar”]
```

**Review by shafreenAnfar — DISMISSED** [review]

File: .github/workflows/invoke-load-tests.yml (hunk 2 above)
Comment (kaneeldias) [reply .github/workflows/invoke-load-tests.yml]: Fixed in 1f1ce36

File: .github/workflows/invoke-load-tests.yml (hunk 1 above)
Comment (kaneeldias) [reply .github/workflows/invoke-load-tests.yml]: Fixed in 1f1ce36

**Review by niveathika — APPROVED** [review]

**Pull request merged**